composer.json

```diff
@@ -163,7 +163,7 @@
             "@check-security",
             "@check-tests"
         ],
-        "check-ecs": "vendor/bin/ecs check packages --clear-cache",
+        "check-ecs": "vendor/bin/ecs check --clear-cache",
```

Comment:
This argument hear silently ignored the `ecs.php` paths setup. 
Using only `ecs.php` is best practise as it's hard to forget all the paths and there is 1 place to change them

Reply:
@TomasVotruba ok thanks for the tip